ledger/ledger-consensual/src/test/java/net/corda/v5/ledger/consensual/ConsensualStatesJavaApiTest.java

```diff
@@ -0,0 +1,48 @@
+package net.corda.v5.ledger.consensual;
+
+import org.assertj.core.api.Assertions;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ConsensualStatesJavaApiTest {
+    private final Party Party = mock(Party.class);
+    private final List<Party> participants = List.of(Party);
+
+    @Test
+    public void participants() {
+
+        final ConsensualState ConsensualState = mock(ConsensualState.class);
+        when(ConsensualState.getParticipants()).thenReturn(participants);
+
+        final List<Party> participants1 = ConsensualState.getParticipants();
+
+        Assertions.assertThat(participants).isEqualTo(participants1);
+    }
+
+    @Test
+    public void customConsensualStateClass() {
+        final CustomConsensualState customConsensualState = new CustomConsensualState(participants);
+
+        Assertions.assertThat(customConsensualState).isNotNull();
+        Assertions.assertThat(customConsensualState.getParticipants()).isEqualTo(participants);
+    }
```

Comment:
Seems like we're just testing the mocks here.

Reply:
I have not dug yet too deeply in the tests. I ported what I found in the old code. It is possible that with the later it became pointless... :) I'll check it later.